ios/ble/central/Central.swift

```diff
@@ -70,4 +68,14 @@ class Central: NSObject, CBCentralManagerDelegate {
             connectedPeripheral.writeValue(messageData, for: characteristic, type: .withoutResponse)
         }
     }
+    
+    func disconnectAndClose() {
```

Comment:
call it just `disconnect` if it is not doing Close, unlike Android.

Reply:
done with suggested changes.